testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -60,6 +62,7 @@ class MiniAppDisplayActivity : BaseActivity() {
 
         if (intent.hasExtra(miniAppTag) || intent.hasExtra(appIdTag)) {
             appId = intent.getStringExtra(appIdTag) ?: ""
+            miniAppInfo = intent.extras?.getParcelable(miniAppTag)!!
```

Comment:
`MiniAppDisplayActivity` is used for two cases: loading with `MiniAppInfo` obj && loading by id. You need to rework on this to ensure it worked on loading by id case.